runtime/compiler/optimizer/J9EstimateCodeSize.cpp

```diff
@@ -1831,7 +1831,7 @@ TR_J9EstimateCodeSize::realEstimateCodeSize(TR_CallTarget *calltarget, TR_CallSt
 
       /*************** PHASE 3:  Optimistically Assume we can partially inline calltarget and add to an optimisticSize ******************/
 
-      calltarget->_cfg = &cfg;
+      calltarget->_cfg = cfgPtr;
```

Comment:
If there aren't any exists between here and where cfgPtr was generated perhaps it would make sense to do this store earlier to save keeping the pointer floating around in a local?